packages/ui-react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -53,7 +54,17 @@ const Dropdown: React.FC<Props & React.AriaAttributes> = ({
         </label>
       )}
       <div className={classNames(styles.dropdown, { [styles.fullWidth]: fullWidth })}>
-        <select id={id} className={styles.select} name={name} value={value} onChange={onChange} aria-required={required} {...rest}>
+        <select
+          id={id}
+          className={styles.select}
+          name={name}
+          value={value}
+          onChange={onChange}
+          aria-required={required}
```

Comment:
`required` is preferred above `aria-required` as argued in my previous comment :)